src/components/Header/Header.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import 'src/components/Header/Header.scss';
+import image from 'public/logo.png';
+
+class Header extends Component {
+  constructor () {
+    super();
+    this.onHeaderMenuClick = this.onHeaderMenuClick.bind(this);
+    this.state = {
+      navIsActive: false
+    };
+  }
+
+  onHeaderMenuClick () {
+    this.setState({
+      navIsActive: !this.state.navIsActive
+    });
+  }
+
+  render () {
+    return (
+      <div className={`header${this.state.navIsActive ? ' is-active' : ''}`}>
```

Comment:
Perhaps we should `BEM` this? Something like `header` if not active or `header--active` when active.